terminology/ansi.py

```diff
@@ -150,6 +155,9 @@ def on_yellow(self) -> "StyledText":
     def underlined(self) -> "StyledText":
         return underlined(self)
 
+    def dimmed(self) -> "StyledText":
+        return dimmed(self)
+
```

Comment:
So, it may seem trivial, but I try to keep my methods and functions alphabetized when there is a lot of them like there are here. There are ways to enforce that, but I haven't set that up. Would you be up for moving `dimmed` to the top